### PR TITLE
Rebalances Crash

### PR DESCRIPTION
**Pull request by blackdav123**
## About The Pull Request
Various changes to make crash more fair.

- Removes hivemind, hivelord, and wraith from crash.
- Xenos get extra t3 slots one xeno later on crash. (see below)

<details> 
  <summary> How many xenos for a t3 slot? </summary>
 
xenopopsheet

</details>

## Why It's Good For The Game
1. Wraith is the bane of this mode and can very easily cheese the Canterbury, this is not fun gameplay and lowpop suffers from having wraith in this mode. Hivemind is completely unkillable on crash and very annoying when it endlessly spams sticky weeds and walls, and hivelord is in a similar position with its large bulk allowing it to spam endless resin and sticky weeds, while also having the healbeam which can give a queen or other large xeno nearly 100% uptime, which is a bit too strong when marines are so short on staff.
2. Marines are not well equipped to deal with t3s on crash and at 7 marines you can already be dealing with two ravagers. This causes very quick wipes and very little back and forth gameplay. Pushing the extra t3 slots to higher pop should help marines survive lowpop easier.

Discuss below what else you think needs tuned, im still not completely sure whether hivelord should stay or go.
## Changelog
:cl:
balance: Removes hivemind, hivelord, and wraith from crash.
balance: Xenos get extra t3 slots one xeno later on crash.

/:cl:

### CODE REVIEW

**Comment by Barnet2** [comment]
Have you decided by Queen, or do you think the rest of the changes will make up for it? Of course, by that point you should be playing Nuclear War, but to preserve the round

**Comment by blackdav123** [comment]
> Have you decided by Queen, or do you think the rest of the changes will make up for it? Of course, by that point you should be playing Nuclear War, but to preserve the round

Queen will probably still be a bit too strong but I couldnt figure out how to do what I wanted, ideally xenos wouldnt gain larva points until the ratio has accounted for the floor of 2 xenos so you'd get your 3rd xeno at 7 marines instead of 4 which would push the queen to be at higher pop than it currently is.

Hopefully removing the hivelord + queen combo will put a decent dent in the issue.

**Comment by Runian** [comment]
For Wraith, can't you just create an area flag kind of like `MARINE_BASE` that prevents blinking into and then apply it to Canterybury? That should solve Wraiths blinking into and slashing the APC. As for sentries, there is certain distance to place them while on radical mode that prevents cheesing with portals; thus skill issue for marines.

**Comment by Runian** [comment]
Revert of #10567. The About Section and Changelog also doesn't mention how dead & pregnant marines are now excluded from the autobal / periodic larva spawning check.

**Comment by blackdav123** [comment]
> Revert of #10567. The About Section and Changelog also doesn't mention how dead & pregnant marines are now excluded from the autobal / periodic larva spawning check.

Fixed, pregnant marines are now counted (we dont live in distress world anymore) and the check for dead marines now works properly excluding DNR marines instead of excluding robots and people without brains/hearts.

**Comment by Naaanii** [comment]
Holy based.

**Comment by Aporhtonoma** [comment]
>Removes hivemind, hivelord, and wraith from crash.
 HOLY BASED

**Review by Kuro020 — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]

**Comment by Lumipharon** [comment]
I'm watching crash games where it is 6 xenos vs 9 marines. This is uh, extremely fucking terrible.

File: code/datums/gamemodes/crash.dm [hunk 1]
```diff
@@ -194,22 +196,11 @@
 	if(stored_larva)
 		return //No need for respawns
 	var/num_xenos = xeno_hive.get_total_xeno_number() + stored_larva
-	if(!num_xenos)
+	if(num_xenos < 2)
 		xeno_job.add_job_positions(1)
```
Comment (Runian) [inline code/datums/gamemodes/crash.dm:200]: Shouldn't this add job positions adjust according to `num_xenos`? `2 - num_xenos`, perhaps?

**Comment by Runian** [comment]
Is it possible to hide the castes from the hive status menu? It seems a bit odd that the castes appear in there as if there could be an xeno with that caste at some point in the gamemode when obviously there won't be since it is impossible to evolve to.

**Comment by WoodenTucker** [comment]
abyssmal dogshit PR

**Pull request merged**